WordPress/src/main/java/org/wordpress/android/ui/stats/StatsInsightsLatestPostSummaryFragment.java

```diff
@@ -0,0 +1,228 @@
+package org.wordpress.android.ui.stats;
+
+import android.graphics.drawable.Drawable;
+import android.text.Spannable;
+import android.text.SpannableString;
+import android.text.style.ForegroundColorSpan;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.android.volley.VolleyError;
+
+import org.wordpress.android.R;
+import org.wordpress.android.ui.ActivityLauncher;
+import org.wordpress.android.ui.stats.exceptions.StatsError;
+import org.wordpress.android.ui.stats.models.InsightsLatestPostModel;
+import org.wordpress.android.ui.stats.models.PostModel;
+import org.wordpress.android.ui.stats.service.StatsService;
+import org.wordpress.android.util.FormatUtils;
+
+public class StatsInsightsLatestPostSummaryFragment extends StatsAbstractInsightsFragment {
+    public static final String TAG = StatsInsightsLatestPostSummaryFragment.class.getSimpleName();
+
+    @Override
+    public void onEventMainThread(StatsEvents.SectionUpdated event) {
+        if (!isAdded()) {
+            return;
+        }
+
+        // This is just an optimization
+        if (event.mEndPointName != StatsService.StatsEndpointsEnum.INSIGHTS_LATEST_POST_VIEWS &&
+                event.mEndPointName != StatsService.StatsEndpointsEnum.INSIGHTS_LATEST_POST_SUMMARY) {
+            return;
+        }
+
+        if (event.mEndPointName != StatsService.StatsEndpointsEnum.INSIGHTS_LATEST_POST_VIEWS) {
```

Comment:
Unfortunately, to populate this module we need data that arrive from 2 different endpoints, and we need to call them sequentially, since the 2nd call requires the ID of the post (that is retrieved in the 1st call).

If the response arrives from the 1st call, we call `super` that in order calls `updateUI `, if the data arrives from the 2nd calls, we need to merge them with the 1st call data and call  `updateUI `. More or less.